miso.cabal

```diff
@@ -30,7 +30,7 @@ common jsaddle-selector
   if arch(wasm32)
     build-depends:
       jsaddle-wasm
-  else
+  elif !impl(ghcjs)
```

Comment:
We should be able to keep this as `else`, since `jsaddle-warp` on js backends is just a no-op and introduces no new deps

```haskell
run :: Int -> JSM () - >IO ()
run _ m = m
```

Reply:
I figured dropping the dependency completely was simpler, seeing as we're already bothering with Cabal conditionals and CPP anyway for the Wasm stuff, but fair enough, I can revert it